app/next-client-app/components/data-table/index.tsx

```diff
@@ -174,15 +165,7 @@ export function DataTable<TData, TValue>({
                   colSpan={columns.length}
                   className="h-24 text-center"
                 >
-                  {emptyStateMessage ? (
-                    <EmptyState
-                      icon={emptyStateIcon}
-                      title={emptyStateMessage}
-                      description={emptyStateDescription || "No results."}
-                    />
-                  ) : (
-                    "No results."
-                  )}
+                  {/* No results message is now handled by individual pages */}
```

Comment:
Could we keep the `no results` functionality as a default? This provides an un-opinionated away to reuse the component, without having to include the other.